firebase-common/data-collection-tests/data-collection-tests.gradle

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Google LLC
```

Comment:
What are these data collection tests? Are these related to upgrading Gradle, or are they part of something else?

Reply:
Indeed they are part of the upgrade, agp 3.4 changed the way robolectric initializes, as a side-effect they no longer support the deprecated feature `@Config(manifest="SomeOtherManifest.xml")`, so I had to extract these into a dedicated project to be able to configure a different data collection value in the manifest for tests.